phoenix-queryserver-client/pom.xml

```diff
@@ -50,11 +50,6 @@
     <url>http://www.apache.org</url>
   </organization>
 
-  <properties>
-    <top.dir>${project.basedir}/..</top.dir>
```

Comment:
We define top.dir everywhere.
I'm not sure if it is needed here or not, but it's outside the scope of this ticket.